### PR TITLE
feat: add config and logger module

### PR DESCRIPTION
**Pull request by shuaijinchao**

### CODE REVIEW

File: apisix/config.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+socket:
+  file: $env.APISIX_LISTEN_ADDRESS
+
+logging:
+  level: debug
```
Comment (spacewander) [inline apisix/config.yaml:22]: Is `debug` suitable as the default log level?

File: apisix/config.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#
```
Comment (spacewander) [inline apisix/config.yaml:1]: Better to mention this file in the doc

File: apisix/config.yaml (hunk 1 above)
Comment (shuaijinchao) [reply apisix/config.yaml:22]: I think `error` is better as the default log level, what do you think.

File: apisix/config.yaml (hunk 2 above)
Comment (shuaijinchao) [reply apisix/config.yaml:1]: OK

**Comment by shuaijinchao** [comment]
@spacewander please review

File: apisix/config.yaml (hunk 1 above)
Comment (spacewander) [reply apisix/config.yaml]: `warn` would be better? Sometimes we need to tell the user something not relative to an error.

Comment (shuaijinchao) [reply apisix/config.yaml]: done

**Review by spacewander — APPROVED** [review]

**Pull request merged**